### ISSUE TITLE
Training without data augmentation

### ISSUE THREAD

**Issue by huangzehao** [szagoruyko/wide-residual-networks#1]
HI, szagoruyko!

Excellent code for learning practice with torch! And I want to do some experiment without data augmentation.  What should I do? Just set _opt.hflip = false_ and _opt.randomcrop = 0_?

Thank you!

**Comment by szagoruyko** [comment]
thanks, you're right, `hflip=false randomcrop=0` will turn off data augmentation

**Comment by huangzehao** [comment]
Thank you!

**Comment by amrit110** [comment]
Hi very inspiring paper!. I am trying to replicate your results. I find that your discussion on using dropout in residual blocks a bit weird, since the original paper on batch-norm argue that batch-norm makes the network layer outputs stochastic given training examples, which is why it has a regularising effect. I think there is a point you say for residual networks with batch-norm "requires heavy data augmentation" and this was the weird part. I do understand your point that due to increased no. of parameters, you want to avoid overfitting, but maybe the argument in that part can be a bit clearer.
